docs/sources/shared/datasources/tempo-search-traceql.md

```diff
@@ -93,6 +97,36 @@ To add a tag, follow these steps:
 1. Select a value from the **Select value** drop-down. This field is populated based upon the tag.
 1. Optional: Select **+** to add an additional tag.
 
+## Optional: Use Aggregate by
+
+{{% admonition type="warning" %}}
+**Aggregate by** is an [experimental feature](/docs/release-life-cycle/). Engineering and on-call support is not available. Documentation is either limited or not provided outside of code comments. No SLA is provided.
+[Enable the `metricsSummary` feature toggle](/docs/grafana/latest/setup-grafana/configure-grafana/feature-toggles/) in Grafana to use this feature. Contact Grafana Support to enable this feature in Grafana Cloud.
+{{% /admonition %}}
+
+Using **Aggregate by**, you can calculate RED metrics (span count, erroring span count, and latency information) for `kind=server` spans recently sent to Tempo, grouped by one or more attributes.
```

Comment:
```suggestion
Using **Aggregate by**, you can calculate RED metrics (total span count, percent erroring spans, and latency information) for spans of `kind=server` that match your filter criteria, grouped by one or more attributes. 
```

Reply:
@joey-grafana -- is the time selector wired up for this feature? I believe in the API we added support for time ranges, so if you say "give me these metrics over the last 10 minutes" we'll calculate those numbers over that exact range. 

I know we can only look back 1 hour in time. So what happens if I say "give me these metrics over the last 2 hours"? I would expect this to fail. What happens if I say "give me these metrics over a 1 hour period that happened 2 days ago"? I would also expect this to fail.

Reply:
@09jvilla, @mdisibio and I tested this today. The data changed depending upon the selections we chose in the time selector for past 5 minutes, 10 minutes, etc. I was running intro-to-mltp as my test ground.

Reply:
Great! 

Now we have to figure out what happens if the chosen time range includes times older than `(current_time - 1 hour`)!